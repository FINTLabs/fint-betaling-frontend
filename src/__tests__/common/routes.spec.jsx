import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import Routes from '../../common/scaffold/routes';

jest.mock('../../feature/dashboard/dashboard_container', () => () => 'dashboard');
jest.mock('../../feature/payment/payment_container', () => () => 'payment');
jest.mock('../../feature/payment-history/payment_history_container', () => () => 'history');
jest.mock('../../feature/log_out/log_out_container', () => () => 'logout');
jest.mock('../../feature/payment/send-payment-to-invoice/send_to_invoice_container', () => () => 'send');
jest.mock('../../feature/payment/sent-payment-to-external/sent_to_external_container', () => () => 'sents');

describe('Routes', () => {
    it('should match snapshot', () => {
        const tree = renderer.create(
            <BrowserRouter basename="/">
                <Routes/>
            </BrowserRouter>,
        )
            .toJSON();

        expect(tree)
            .toMatchSnapshot();
    });
});
