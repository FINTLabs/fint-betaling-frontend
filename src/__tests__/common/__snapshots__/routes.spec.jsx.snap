// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should match snapshot 1`] = `
<div>
  dashboard
</div>
`;
